assets/package.json

```diff
@@ -3,32 +3,14 @@
   "description": "TIL is an open-source project by Hashrocket that exists to catalogue the sharing & accumulation of knowledge as it happens day-to-day.",
   "license": "MIT",
   "scripts": {
-    "deploy": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
   },
   "dependencies": {
-    "autosize": "^4.0.2",
-    "codemirror": "^5.61.1",
+    "autosize": "^5.0.1",
+    "codemirror": "^5.65.9",
     "codemirror-mode-elixir": "^1.1.1",
-    "jquery": "^3.6.0",
-    "jquery.cookie": "^1.4.1",
-    "phoenix": "file:../deps/phoenix",
-    "phoenix_html": "file:../deps/phoenix_html"
+    "jquery": "^3.6.1",
+    "jquery.cookie": "^1.4.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/plugin-proposal-class-properties": "^7.7.4",
-    "@babel/preset-env": "^7.8.4",
-    "babel-loader": "^8.0.6",
-    "compression-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^5.0.4",
-    "css-loader": "^3.2.0",
-    "mini-css-extract-plugin": "^0.8.0",
-    "sass": "^1.55.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.3",
-    "sass-loader": "10.2.1",
-    "uglifyjs-webpack-plugin": "^2.2.0",
-    "webpack": "4.41.5",
-    "webpack-cli": "^3.3.10"
```

Comment:
Really great to see all of this removed! 

Shouldn't we include esbuild as dev dependency? If not, we can just remove the `devDependencies` entirley

Reply:
the `esbuild` hex package executes its own encapsulated esbuild via a mix task, so we don't need to install that via npm. I deleted the `devDependencies` thanks!
As an additional note, I tried to make this PR to be smaller than that, my initial plans was to leave the esbuild migration to another PR, but when I bumped some of the phoenix hex deps, I think that liveview, their new js version was breaking webpacker for some reason. The error message was not great to find out a solution, so I end up pushing for migrating that to esbuild in this same huge PR.